guides/contributor/genai.md

```diff
@@ -0,0 +1,91 @@
+# Generative AI Contribution Policy
+
+This policy provides general guidance for contributors and maintainers relating
+to the use of Generative AI in OpenTelemetry projects. This guidance supersedes
+and extends the policy defined in the [Linux Foundation Generative AI
+Policy](https://www.linuxfoundation.org/legal/generative-ai).
+
+## The Short Version
+
+While we welcome contributions from anyone, maintainers of individual
+projects may -- at their discretion -- hide or close issues, pull requests, or
+other contributions that are made totally or in part through generative AI
+tooling.
+
+## The Long Version
+
+Increasingly, we have observed a trend of contributors who are utilizing LLMs
+and other generative tools to participate in issues and create pull requests.
+Regurgitating the output of an LLM is unlikely to be particularly helpful, or
+valuable, to other contributors, maintainers, and end-users. OpenTelemetry is a
```

Comment:
I think it would be helpful to elaborate on this a bit. My 2 cents:

Time is a scarce resource. Thoughtful code reviews are very time consuming. There just isn't enough time to give proper responses to a flood of low-effort, low-quality contributions without compromising responsiveness to high-effort, high quality contributions. We need to prioritize the quality contributions to keep the ecosystem healthy.

No need to include this, but you're welcome to if any of it resonates. I'm approving as is.

Reply:
Approving as well, but +1 on this comment